packages/api/src/beacon/routes/beacon/index.ts

```diff
@@ -28,51 +26,41 @@ export type {
   StateId,
   ValidatorId,
   ValidatorStatus,
-  ValidatorFilters,
-  CommitteesFilters,
   FinalityCheckpoints,
   ValidatorResponse,
   ValidatorBalance,
   EpochCommitteeResponse,
   EpochSyncCommitteeResponse,
 } from "./state.js";
 
-export type Api = block.Api &
-  pool.Api &
-  state.Api &
-  rewards.Api & {
-    getGenesis(): Promise<ApiClientResponse<{[HttpStatusCode.OK]: {data: phase0.Genesis}}>>;
+export type Endpoints = block.Endpoints &
+  pool.Endpoints &
+  state.Endpoints &
+  rewards.Endpoints & {
+    getGenesis: Endpoint<
+      //
```

Comment:
```suggestion
```

Reply:
The `//` is added here to force a line break, otherwise `Endpoint` will be formatted as a single line which makes it harder to read

Reply:
I would recommend using what's default formatting enforces, otherwise no value having it. 

In case you still prefer not to then use `/* prettier-ignore */` instead of empty line comments that's confusing.

Reply:
I find the prettier ignore a bit distracting when reading code, also adding something like `comment for formatting` feels too much as well.

To make it little distracting but at the same time clear that the comment is there intentionally, I replaced all empty comment with `// ⏎`. We use that in other places in Lodestar already and I find it's a neat solution / good middle ground.